# HG changeset patch
# Parent 9fcd533086651c2be4a607929019e3a2b50ed03d
# User Frédéric Wang <fred.wang@free.fr>
MathML Operator Dictionary: fix space conflicts (bug 534970). r=karlt

diff --git a/layout/mathml/mathfont.properties b/layout/mathml/mathfont.properties
--- a/layout/mathml/mathfont.properties
+++ b/layout/mathml/mathfont.properties
@@ -114,61 +114,61 @@ mathvariant.bold-sans-serif = A\ud835\ud
 mathvariant.sans-serif-italic = A\ud835\ude08B\ud835\ude09C\ud835\ude0aD\ud835\ude0bE\ud835\ude0cF\ud835\ude0dG\ud835\ude0eH\ud835\ude0fI\ud835\ude10J\ud835\ude11K\ud835\ude12L\ud835\ude13M\ud835\ude14N\ud835\ude15O\ud835\ude16P\ud835\ude17Q\ud835\ude18R\ud835\ude19S\ud835\ude1aT\ud835\ude1bU\ud835\ude1cV\ud835\ude1dW\ud835\ude1eX\ud835\ude1fY\ud835\ude20Z\ud835\ude21a\ud835\ude22b\ud835\ude23c\ud835\ude24d\ud835\ude25e\ud835\ude26f\ud835\ude27g\ud835\ude28h\ud835\ude29i\ud835\ude2aj\ud835\ude2bk\ud835\ude2cl\ud835\ude2dm\ud835\ude2en\ud835\ude2fo\ud835\ude30p\ud835\ude31q\ud835\ude32r\ud835\ude33s\ud835\ude34t\ud835\ude35u\ud835\ude36v\ud835\ude37w\ud835\ude38x\ud835\ude39y\ud835\ude3az\ud835\ude3b
 mathvariant.sans-serif-bold-italic = A\ud835\ude3cB\ud835\ude3dC\ud835\ude3eD\ud835\ude3fE\ud835\ude40F\ud835\ude41G\ud835\ude42H\ud835\ude43I\ud835\ude44J\ud835\ude45K\ud835\ude46L\ud835\ude47M\ud835\ude48N\ud835\ude49O\ud835\ude4aP\ud835\ude4bQ\ud835\ude4cR\ud835\ude4dS\ud835\ude4eT\ud835\ude4fU\ud835\ude50V\ud835\ude51W\ud835\ude52X\ud835\ude53Y\ud835\ude54Z\ud835\ude55a\ud835\ude56b\ud835\ude57c\ud835\ude58d\ud835\ude59e\ud835\ude5af\ud835\ude5bg\ud835\ude5ch\ud835\ude5di\ud835\ude5ej\ud835\ude5fk\ud835\ude60l\ud835\ude61m\ud835\ude62n\ud835\ude63o\ud835\ude64p\ud835\ude65q\ud835\ude66r\ud835\ude67s\ud835\ude68t\ud835\ude69u\ud835\ude6av\ud835\ude6bw\ud835\ude6cx\ud835\ude6dy\ud835\ude6ez\ud835\ude6f\u0391\ud835\udf90\u0392\ud835\udf91\u0393\ud835\udf92\u0394\ud835\udf93\u0395\ud835\udf94\u0396\ud835\udf95\u0397\ud835\udf96\u0398\ud835\udf97\u0399\ud835\udf98\u039A\ud835\udf99\u039B\ud835\udf9a\u039C\ud835\udf9b\u039D\ud835\udf9c\u039E\ud835\udf9d\u039F\ud835\udf9e\u03A0\ud835\udf9f\u03A1\ud835\udfa0\u03F4\ud835\udfa1\u03A3\ud835\udfa2\u03A4\ud835\udfa3\u03A5\ud835\udfa4\u03A6\ud835\udfa5\u03A7\ud835\udfa6\u03A8\ud835\udfa7\u03A9\ud835\udfa8\u2207\ud835\udfa9\u03B1\ud835\udfaa\u03B2\ud835\udfab\u03B3\ud835\udfac\u03B4\ud835\udfad\u03B5\ud835\udfae\u03B6\ud835\udfaf\u03B7\ud835\udfb0\u03B8\ud835\udfb1\u03B9\ud835\udfb2\u03BA\ud835\udfb3\u03BB\ud835\udfb4\u03BC\ud835\udfb5\u03BD\ud835\udfb6\u03BE\ud835\udfb7\u03BF\ud835\udfb8\u03C0\ud835\udfb9\u03C1\ud835\udfba\u03C2\ud835\udfbb\u03C3\ud835\udfbc\u03C4\ud835\udfbd\u03C5\ud835\udfbe\u03C6\ud835\udfbf\u03C7\ud835\udfc0\u03C8\ud835\udfc1\u03C9\ud835\udfc2\u2202\ud835\udfc3\u220A\ud835\udfc4\u03D1\ud835\udfc5\u03F0\ud835\udfc6\u03D5\ud835\udfc7\u03F1\ud835\udfc8\u03D6\ud835\udfc9
 mathvariant.monospace = A\ud835\ude70B\ud835\ude71C\ud835\ude72D\ud835\ude73E\ud835\ude74F\ud835\ude75G\ud835\ude76H\ud835\ude77I\ud835\ude78J\ud835\ude79K\ud835\ude7aL\ud835\ude7bM\ud835\ude7cN\ud835\ude7dO\ud835\ude7eP\ud835\ude7fQ\ud835\ude80R\ud835\ude81S\ud835\ude82T\ud835\ude83U\ud835\ude84V\ud835\ude85W\ud835\ude86X\ud835\ude87Y\ud835\ude88Z\ud835\ude89a\ud835\ude8ab\ud835\ude8bc\ud835\ude8cd\ud835\ude8de\ud835\ude8ef\ud835\ude8fg\ud835\ude90h\ud835\ude91i\ud835\ude92j\ud835\ude93k\ud835\ude94l\ud835\ude95m\ud835\ude96n\ud835\ude97o\ud835\ude98p\ud835\ude99q\ud835\ude9ar\ud835\ude9bs\ud835\ude9ct\ud835\ude9du\ud835\ude9ev\ud835\ude9fw\ud835\udea0x\ud835\udea1y\ud835\udea2z\ud835\udea30\ud835\udff61\ud835\udff72\ud835\udff83\ud835\udff94\ud835\udffa5\ud835\udffb6\ud835\udffc7\ud835\udffd8\ud835\udffe9\ud835\udfff
 
 
 # Operator Dictionary indexed on the "form" (i.e., infix, prefix, or suffix).
 # Each entry lists the attributes of the operator, using its Unicode format.
 
-operator.\u0021.postfix = lspace:2 rspace:0 # !
-operator.\u0021\u0021.postfix = lspace:2 rspace:0 # !!
-operator.\u0021\u003D.infix = lspace:5 rspace:5 # !=
+operator.\u0021.postfix = lspace:1 rspace:0 # !
+operator.\u0021\u0021.postfix = lspace:1 rspace:0 # !!
+operator.\u0021\u003D.infix = lspace:4 rspace:4 # !=
 operator.\u0025.infix = lspace:3 rspace:3 # percent sign
-operator.\u0026.postfix = lspace:5 rspace:0 # &amp;
-operator.\u0026\u0026.infix = lspace:5 rspace:5 # &amp;&amp;
+operator.\u0026.postfix = lspace:0 rspace:0 # &amp;
+operator.\u0026\u0026.infix = lspace:4 rspace:4 # &amp;&amp;
 operator.\u0027.postfix = lspace:2 rspace:0 # '
 operator.\u0028.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # (
 operator.\u0029.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # )
 operator.\u002A.infix = lspace:3 rspace:3 # *
-operator.\u002A\u002A.infix = lspace:2 rspace:2 # **
-operator.\u002A\u003D.infix = lspace:5 rspace:5 # *=
+operator.\u002A\u002A.infix = lspace:1 rspace:1 # **
+operator.\u002A\u003D.infix = lspace:4 rspace:4 # *=
 operator.\u002B.infix = lspace:4 rspace:4 # +
 operator.\u002B.prefix = lspace:0 rspace:1 # +
-operator.\u002B\u002B.postfix = lspace:2 rspace:0 # ++
-operator.\u002B\u003D.infix = lspace:5 rspace:5 # +=
-operator.\u002C.infix = lspace:0 rspace:6 separator # ,
+operator.\u002B\u002B.postfix = lspace:0 rspace:0 # ++
+operator.\u002B\u003D.infix = lspace:4 rspace:4 # +=
+operator.\u002C.infix = lspace:0 rspace:3 separator # ,
 operator.\u002D.infix = lspace:4 rspace:4 # -
 operator.\u002D.prefix = lspace:0 rspace:1 # -
-operator.\u002D\u002D.postfix = lspace:2 rspace:0 # --
-operator.\u002D\u003D.infix = lspace:5 rspace:5 # -=
+operator.\u002D\u002D.postfix = lspace:0 rspace:0 # --
+operator.\u002D\u003D.infix = lspace:4 rspace:4 # -=
 operator.\u002D\u003E.infix = lspace:5 rspace:5 # ->
-operator.\u002E.infix = lspace:0 rspace:0 # .
-operator.\u002E\u002E.postfix = lspace:4 rspace:0 # ..
-operator.\u002E\u002E\u002E.postfix = lspace:4 rspace:0 # ...
+operator.\u002E.infix = lspace:3 rspace:3 # .
+operator.\u002E\u002E.postfix = lspace:0 rspace:0 # ..
+operator.\u002E\u002E\u002E.postfix = lspace:0 rspace:0 # ...
 operator.\u002F.infix = lspace:3 rspace:3 stretchy direction:vertical # /
-operator.\u002F\u002F.infix = lspace:5 rspace:5 # //
-operator.\u002F\u003D.infix = lspace:5 rspace:5 # /=
-operator.\u003A.infix = lspace:5 rspace:5 # :
-operator.\u003A\u003D.infix = lspace:5 rspace:5 # :=
-operator.\u003B.infix = lspace:0 rspace:5 separator # ;
+operator.\u002F\u002F.infix = lspace:1 rspace:1 # //
+operator.\u002F\u003D.infix = lspace:4 rspace:4 # /=
+operator.\u003A.infix = lspace:1 rspace:2 # :
+operator.\u003A\u003D.infix = lspace:4 rspace:4 # :=
+operator.\u003B.infix = lspace:0 rspace:3 separator # ;
 operator.\u003C.infix = lspace:5 rspace:5 # &lt;
 operator.\u003C\u003D.infix = lspace:5 rspace:5 # &lt;=
-operator.\u003C\u003E.infix = lspace:2 rspace:2 # &lt;>
+operator.\u003C\u003E.infix = lspace:1 rspace:1 # &lt;>
 operator.\u003D.infix = lspace:5 rspace:5 # =
-operator.\u003D\u003D.infix = lspace:5 rspace:5 # ==
+operator.\u003D\u003D.infix = lspace:4 rspace:4 # ==
 operator.\u003E.infix = lspace:5 rspace:5 # >
 operator.\u003E\u003D.infix = lspace:5 rspace:5 # >=
-operator.\u003F.infix = lspace:2 rspace:2 # ?
-operator.\u0040.infix = lspace:2 rspace:2 # @
+operator.\u003F.infix = lspace:1 rspace:1 # ?
+operator.\u0040.infix = lspace:1 rspace:1 # @
 operator.\u005B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # [
 operator.\u005C.infix = lspace:0 rspace:0 # reverse solidus
 operator.\u005D.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # ]
 operator.\u005E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &Hat; circumflex accent
-operator.\u005E.infix = lspace:2 rspace:2 direction:horizontal # ^
+operator.\u005E.infix = lspace:1 rspace:1 direction:horizontal # ^
 operator.\u005F.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # _ low line
 operator.\u005F.infix = lspace:2 rspace:2 stretchy direction:horizontal # _
 operator.\u0060.postfix = lspace:0 rspace:0 accent # &DiacriticalGrave;
 operator.\u007B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # {
 operator.\u007C.infix = lspace:0 rspace:0 stretchy symmetric direction:vertical # &VerticalLine; |
 operator.\u007C.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
 operator.\u007C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
 operator.\u007C\u007C.infix = lspace:4 rspace:4 # ||
@@ -176,17 +176,17 @@ operator.\u007D.postfix = lspace:0 rspac
 operator.\u007E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # ~ tilde
 operator.\u00A8.postfix = lspace:0 rspace:0 accent # &DoubleDot;
 operator.\u00AC.prefix = lspace:2 rspace:1 # not sign
 operator.\u00AF.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverBar;
 operator.\u00B0.postfix = lspace:0 rspace:0 # degree sign
 operator.\u00B1.infix = lspace:4 rspace:4 # &PlusMinus;
 operator.\u00B1.prefix = lspace:0 rspace:1 # &PlusMinus;
 operator.\u00B4.postfix = lspace:0 rspace:0 accent # &DiacriticalAcute;
-operator.\u00B7.infix = lspace:3 rspace:3 # &CenterDot;
+operator.\u00B7.infix = lspace:4 rspace:4 # &CenterDot;
 operator.\u00B8.postfix = lspace:0 rspace:0 accent # &Cedilla;
 operator.\u00D7.infix = lspace:4 rspace:4 # multiplication sign
 operator.\u00F7.infix = lspace:4 rspace:4 # division sign
 operator.\u02C6.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # modifier letter circumflex accent
 operator.\u02C7.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &Hacek; caron
 operator.\u02CA.postfix = lspace:0 rspace:0 accent # modifier letter acute accent
 operator.\u02CB.postfix = lspace:0 rspace:0 accent # modifier letter grave accent
 operator.\u02D8.postfix = lspace:0 rspace:0 accent # &Breve;
@@ -207,42 +207,42 @@ operator.\u2026.infix = lspace:0 rspace:
 operator.\u2032.postfix = lspace:0 rspace:2 # prime
 operator.\u203E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # overline
 operator.\u2061.infix = lspace:0 rspace:0 # &ApplyFunction;
 operator.\u2062.infix = lspace:0 rspace:0 # &InvisibleTimes;
 operator.\u2063.infix = lspace:0 rspace:0 separator # &InvisibleComma;
 operator.\u2064.infix = lspace:0 rspace:0 # invisible plus
 operator.\u20DB.postfix = lspace:0 rspace:0 accent # &TripleDot;
 operator.\u20DC.postfix = lspace:0 rspace:0 accent # combining four dots above
-operator.\u2145.prefix = lspace:0 rspace:2 # &CapitalDifferentialD;
-operator.\u2146.prefix = lspace:0 rspace:2 # &DifferentialD;
+operator.\u2145.prefix = lspace:2 rspace:1 # &CapitalDifferentialD;
+operator.\u2146.prefix = lspace:2 rspace:0 # &DifferentialD;
 operator.\u2190.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrow;
-operator.\u2191.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrow;
+operator.\u2191.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrow;
 operator.\u2192.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrow;
-operator.\u2193.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrow;
+operator.\u2193.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrow;
 operator.\u2194.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightArrow;
-operator.\u2195.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpDownArrow;
+operator.\u2195.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpDownArrow;
 operator.\u2196.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperLeftArrow;
 operator.\u2197.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperRightArrow;
 operator.\u2198.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerRightArrow;
 operator.\u2199.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerLeftArrow;
 operator.\u219A.infix = lspace:5 rspace:5 accent # leftwards arrow with stroke
 operator.\u219B.infix = lspace:5 rspace:5 accent # rightwards arrow with stroke
 operator.\u219C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards wave arrow
 operator.\u219D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards wave arrow
 operator.\u219E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards two headed arrow
 operator.\u219F.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards two headed arrow
 operator.\u21A0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two headed arrow
 operator.\u21A1.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards two headed arrow
 operator.\u21A2.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with tail
 operator.\u21A3.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with tail
 operator.\u21A4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeArrow;
-operator.\u21A5.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpTeeArrow;
+operator.\u21A5.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpTeeArrow;
 operator.\u21A6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeArrow;
-operator.\u21A7.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownTeeArrow;
+operator.\u21A7.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownTeeArrow;
 operator.\u21A8.infix = lspace:5 rspace:5 stretchy direction:vertical # up down arrow with base
 operator.\u21A9.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookleftarrow; &larrhk;
 operator.\u21AA.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookrightarrow; &rarrhk;
 operator.\u21AB.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with loop
 operator.\u21AC.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with loop
 operator.\u21AD.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right wave arrow
 operator.\u21AE.infix = lspace:5 rspace:5 accent # left right arrow with stroke
 operator.\u21AF.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards zigzag arrow
@@ -255,40 +255,40 @@ operator.\u21B5.infix = lspace:5 rspace:
 operator.\u21B6.infix = lspace:5 rspace:5 accent # anticlockwise top semicircle arrow
 operator.\u21B7.infix = lspace:5 rspace:5 accent # clockwise top semicircle arrow
 operator.\u21B8.infix = lspace:5 rspace:5 # north west arrow to long bar
 operator.\u21B9.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow to bar over rightwards arrow to bar
 operator.\u21BA.infix = lspace:5 rspace:5 # anticlockwise open circle arrow
 operator.\u21BB.infix = lspace:5 rspace:5 # clockwise open circle arrow
 operator.\u21BC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVector;
 operator.\u21BD.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVector;
-operator.\u21BE.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpVector;
-operator.\u21BF.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpVector;
+operator.\u21BE.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpVector;
+operator.\u21BF.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpVector;
 operator.\u21C0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVector;
 operator.\u21C1.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVector;
-operator.\u21C2.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownVector;
-operator.\u21C3.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownVector;
+operator.\u21C2.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownVector;
+operator.\u21C3.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownVector;
 operator.\u21C4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrowLeftArrow;
-operator.\u21C5.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrowDownArrow;
+operator.\u21C5.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrowDownArrow;
 operator.\u21C6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrowRightArrow;
 operator.\u21C7.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards paired arrows
 operator.\u21C8.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards paired arrows
 operator.\u21C9.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards paired arrows
 operator.\u21CA.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards paired arrows
 operator.\u21CB.infix = lspace:5 rspace:5 stretchy direction:horizontal # &ReverseEquilibrium;
 operator.\u21CC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Equilibrium;
 operator.\u21CD.infix = lspace:5 rspace:5 accent # leftwards double arrow with stroke
 operator.\u21CE.infix = lspace:5 rspace:5 accent # left right double arrow with stroke
 operator.\u21CF.infix = lspace:5 rspace:5 accent # rightwards double arrow with stroke
 operator.\u21D0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftArrow;
-operator.\u21D1.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleUpArrow;
+operator.\u21D1.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleUpArrow;
 operator.\u21D2.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Implies; &DoubleRightArrow;
-operator.\u21D3.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleDownArrow;
+operator.\u21D3.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleDownArrow;
 operator.\u21D4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftRightArrow;
-operator.\u21D5.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleUpDownArrow;
+operator.\u21D5.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleUpDownArrow;
 operator.\u21DA.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple arrow
 operator.\u21DB.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple arrow
 operator.\u21DC.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards squiggle arrow
 operator.\u21DD.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards squiggle arrow
 operator.\u21DE.infix = lspace:5 rspace:5 # upwards arrow with double stroke
 operator.\u21DF.infix = lspace:5 rspace:5 # downwards arrow with double stroke
 operator.\u21E0.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards dashed arrow
 operator.\u21E1.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards dashed arrow
@@ -306,34 +306,34 @@ operator.\u21EC.infix = lspace:5 rspace:
 operator.\u21ED.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal with vertical bar
 operator.\u21EE.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow
 operator.\u21EF.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow on pedestal
 operator.\u21F0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards white arrow from wall
 operator.\u21F1.infix = lspace:5 rspace:5 # north west arrow to corner
 operator.\u21F2.infix = lspace:5 rspace:5 # south east arrow to corner
 operator.\u21F3.infix = lspace:5 rspace:5 stretchy direction:vertical # up down white arrow
 operator.\u21F4.infix = lspace:5 rspace:5 accent # right arrow with small circle
-operator.\u21F5.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrowUpArrow;
+operator.\u21F5.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrowUpArrow;
 operator.\u21F6.infix = lspace:5 rspace:5 stretchy direction:horizontal # three rightwards arrows
 operator.\u21F7.infix = lspace:5 rspace:5 accent # leftwards arrow with vertical stroke
 operator.\u21F8.infix = lspace:5 rspace:5 accent # rightwards arrow with vertical stroke
 operator.\u21F9.infix = lspace:5 rspace:5 accent # left right arrow with vertical stroke
 operator.\u21FA.infix = lspace:5 rspace:5 accent # leftwards arrow with double vertical stroke
 operator.\u21FB.infix = lspace:5 rspace:5 accent # rightwards arrow with double vertical stroke
 operator.\u21FC.infix = lspace:5 rspace:5 accent # left right arrow with double vertical stroke
 operator.\u21FD.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards open-headed arrow
 operator.\u21FE.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards open-headed arrow
 operator.\u21FF.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right open-headed arrow
-operator.\u2200.prefix = lspace:0 rspace:5 # &ForAll;
+operator.\u2200.prefix = lspace:2 rspace:1 # &ForAll;
 operator.\u2201.infix = lspace:1 rspace:2 # complement
-operator.\u2202.prefix = lspace:0 rspace:2 # &PartialD;
-operator.\u2203.prefix = lspace:0 rspace:5 # &Exists;
-operator.\u2204.prefix = lspace:0 rspace:5 # &NotExists;
+operator.\u2202.prefix = lspace:2 rspace:1 # &PartialD;
+operator.\u2203.prefix = lspace:2 rspace:1 # &Exists;
+operator.\u2204.prefix = lspace:2 rspace:1 # &NotExists;
 operator.\u2206.infix = lspace:3 rspace:3 # increment
-operator.\u2207.prefix = lspace:0 rspace:2 # &Del;
+operator.\u2207.prefix = lspace:2 rspace:1 # &Del;
 operator.\u2208.infix = lspace:5 rspace:5 # &Element;
 operator.\u2209.infix = lspace:5 rspace:5 # &NotElement;
 operator.\u220A.infix = lspace:5 rspace:5 # small element of
 operator.\u220B.infix = lspace:5 rspace:5 # &SuchThat; &ReverseElement;
 operator.\u220C.infix = lspace:5 rspace:5 # &NotReverseElement;
 operator.\u220D.infix = lspace:5 rspace:5 # small contains as member
 operator.\u220E.infix = lspace:3 rspace:3 # end of proof
 operator.\u220F.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Product;
@@ -341,19 +341,19 @@ operator.\u2210.prefix = lspace:1 rspace
 operator.\u2211.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Sum;
 operator.\u2212.infix = lspace:4 rspace:4 # official Unicode minus sign
 operator.\u2212.prefix = lspace:0 rspace:1 # official Unicode minus sign
 operator.\u2213.infix = lspace:4 rspace:4 # &MinusPlus;
 operator.\u2213.prefix = lspace:0 rspace:1 # &MinusPlus;
 operator.\u2214.infix = lspace:4 rspace:4 # dot plus
 operator.\u2216.infix = lspace:3 rspace:3 stretchy direction:vertical # &Backslash;
 operator.\u2217.infix = lspace:4 rspace:4 # asterisk operator
-operator.\u2218.infix = lspace:2 rspace:2 # &SmallCircle;
+operator.\u2218.infix = lspace:4 rspace:4 # &SmallCircle;
 operator.\u2219.infix = lspace:4 rspace:4 # bullet operator
-operator.\u221A.prefix = lspace:0 rspace:2 stretchy direction:vertical # &Sqrt;
+operator.\u221A.prefix = lspace:1 rspace:1 stretchy direction:vertical # &Sqrt;
 operator.\u221B.prefix = lspace:1 rspace:1 # cube root
 operator.\u221C.prefix = lspace:1 rspace:1 # fourth root
 operator.\u221D.infix = lspace:5 rspace:5 # &Proportional;
 operator.\u221F.infix = lspace:5 rspace:5 # right angle
 operator.\u2220.prefix = lspace:0 rspace:0 # angle
 operator.\u2221.prefix = lspace:0 rspace:0 # measured angle
 operator.\u2222.prefix = lspace:0 rspace:0 # spherical angle
 operator.\u2223.infix = lspace:5 rspace:5 stretchy direction:vertical # &VerticalBar;
@@ -381,17 +381,17 @@ operator.\u2238.infix = lspace:4 rspace:
 operator.\u2239.infix = lspace:5 rspace:5 # excess
 operator.\u223A.infix = lspace:4 rspace:4 # geometric proportion
 operator.\u223B.infix = lspace:5 rspace:5 # homothetic
 operator.\u223C.infix = lspace:5 rspace:5 # &Tilde;
 operator.\u223D.infix = lspace:5 rspace:5 # reversed tilde
 operator.\u223D\u0331.infix = lspace:3 rspace:3 # reversed tilde with underline
 operator.\u223E.infix = lspace:5 rspace:5 # inverted lazy s
 operator.\u223F.infix = lspace:3 rspace:3 # sine wave
-operator.\u2240.infix = lspace:3 rspace:3 # &VerticalTilde;
+operator.\u2240.infix = lspace:4 rspace:4 # &VerticalTilde;
 operator.\u2241.infix = lspace:5 rspace:5 # &NotTilde;
 operator.\u2242.infix = lspace:5 rspace:5 # &EqualTilde;
 operator.\u2242\u0338.infix = lspace:5 rspace:5 # &NotEqualTilde;
 operator.\u2243.infix = lspace:5 rspace:5 # &TildeEqual;
 operator.\u2244.infix = lspace:5 rspace:5 # &NotTildeEqual;
 operator.\u2245.infix = lspace:5 rspace:5 # &TildeFullEqual;
 operator.\u2246.infix = lspace:5 rspace:5 # approximately but not actually equal to
 operator.\u2247.infix = lspace:5 rspace:5 # &NotTildeFullEqual;
@@ -476,21 +476,21 @@ operator.\u228E.infix = lspace:4 rspace:
 operator.\u228F.infix = lspace:5 rspace:5 # &SquareSubset;
 operator.\u228F\u0338.infix = lspace:5 rspace:5 # &NotSquareSubset;
 operator.\u2290.infix = lspace:5 rspace:5 # &SquareSuperset;
 operator.\u2290\u0338.infix = lspace:5 rspace:5 # &NotSquareSuperset;
 operator.\u2291.infix = lspace:5 rspace:5 # &SquareSubsetEqual;
 operator.\u2292.infix = lspace:5 rspace:5 # &SquareSupersetEqual;
 operator.\u2293.infix = lspace:4 rspace:4 direction:vertical # &SquareIntersection;
 operator.\u2294.infix = lspace:4 rspace:4 direction:vertical # &SquareUnion;
-operator.\u2295.infix = lspace:3 rspace:3 # &CirclePlus;
-operator.\u2296.infix = lspace:3 rspace:3 # &CircleMinus;
-operator.\u2297.infix = lspace:3 rspace:3 # &CircleTimes;
+operator.\u2295.infix = lspace:4 rspace:4 # &CirclePlus;
+operator.\u2296.infix = lspace:4 rspace:4 # &CircleMinus;
+operator.\u2297.infix = lspace:4 rspace:4 # &CircleTimes;
 operator.\u2298.infix = lspace:4 rspace:4 # circled division slash
-operator.\u2299.infix = lspace:2 rspace:2 # &CircleDot;
+operator.\u2299.infix = lspace:4 rspace:4 # &CircleDot;
 operator.\u229A.infix = lspace:4 rspace:4 # circled ring operator
 operator.\u229B.infix = lspace:4 rspace:4 # circled asterisk operator
 operator.\u229C.infix = lspace:4 rspace:4 # circled equals
 operator.\u229D.infix = lspace:4 rspace:4 # circled dash
 operator.\u229E.infix = lspace:4 rspace:4 # squared plus
 operator.\u229F.infix = lspace:4 rspace:4 # squared minus
 operator.\u22A0.infix = lspace:4 rspace:4 # squared times
 operator.\u22A1.infix = lspace:4 rspace:4 # squared dot operator
@@ -522,33 +522,33 @@ operator.\u22BA.infix = lspace:4 rspace:
 operator.\u22BB.infix = lspace:4 rspace:4 # xor
 operator.\u22BC.infix = lspace:4 rspace:4 # nand
 operator.\u22BD.infix = lspace:4 rspace:4 # nor
 operator.\u22BE.infix = lspace:3 rspace:3 # right angle with arc
 operator.\u22BF.infix = lspace:3 rspace:3 # right triangle
 operator.\u22C0.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Wedge;
 operator.\u22C1.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Vee;
 operator.\u22C2.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Intersection;
-operator.\u22C3.prefix = lspace:0 rspace:3 largeop movablelimits symmetric direction:vertical # &Union;
-operator.\u22C4.infix = lspace:3 rspace:3 # &Diamond;
-operator.\u22C5.infix = lspace:3 rspace:3 # &cdot;
-operator.\u22C6.infix = lspace:3 rspace:3 # &Star;
+operator.\u22C3.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Union;
+operator.\u22C4.infix = lspace:4 rspace:4 # &Diamond;
+operator.\u22C5.infix = lspace:4 rspace:4 # &cdot;
+operator.\u22C6.infix = lspace:4 rspace:4 # &Star;
 operator.\u22C7.infix = lspace:4 rspace:4 # division times
 operator.\u22C8.infix = lspace:5 rspace:5 # bowtie
 operator.\u22C9.infix = lspace:4 rspace:4 # left normal factor semidirect product
 operator.\u22CA.infix = lspace:4 rspace:4 # right normal factor semidirect product
 operator.\u22CB.infix = lspace:4 rspace:4 # left semidirect product
 operator.\u22CC.infix = lspace:4 rspace:4 # right semidirect product
 operator.\u22CD.infix = lspace:5 rspace:5 # reversed tilde equals
 operator.\u22CE.infix = lspace:4 rspace:4 # curly logical or
 operator.\u22CF.infix = lspace:4 rspace:4 # curly logical and
 operator.\u22D0.infix = lspace:5 rspace:5 # &Subset;
 operator.\u22D1.infix = lspace:5 rspace:5 # double superset
-operator.\u22D2.infix = lspace:3 rspace:3 # &Cap;
-operator.\u22D3.infix = lspace:3 rspace:3 # &Cup;
+operator.\u22D2.infix = lspace:4 rspace:4 # &Cap;
+operator.\u22D3.infix = lspace:4 rspace:4 # &Cup;
 operator.\u22D4.infix = lspace:5 rspace:5 # pitchfork
 operator.\u22D5.infix = lspace:5 rspace:5 # equal and parallel to
 operator.\u22D6.infix = lspace:5 rspace:5 # less-than with dot
 operator.\u22D7.infix = lspace:5 rspace:5 # greater-than with dot
 operator.\u22D8.infix = lspace:5 rspace:5 # very much less-than
 operator.\u22D9.infix = lspace:5 rspace:5 # very much greater-than
 operator.\u22DA.infix = lspace:5 rspace:5 # &LessEqualGreater;
 operator.\u22DB.infix = lspace:5 rspace:5 # &GreaterEqualLess;
@@ -663,18 +663,18 @@ operator.\u2909.infix = lspace:5 rspace:
 operator.\u290A.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards triple arrow
 operator.\u290B.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards triple arrow
 operator.\u290C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards double dash arrow
 operator.\u290D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards double dash arrow
 operator.\u290E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple dash arrow
 operator.\u290F.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple dash arrow
 operator.\u2910.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two-headed triple dash arrow
 operator.\u2911.infix = lspace:5 rspace:5 accent # rightwards arrow with dotted stem
-operator.\u2912.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrowBar;
-operator.\u2913.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrowBar;
+operator.\u2912.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrowBar;
+operator.\u2913.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrowBar;
 operator.\u2914.infix = lspace:5 rspace:5 accent # rightwards arrow with tail with vertical stroke
 operator.\u2915.infix = lspace:5 rspace:5 accent # rightwards arrow with tail with double vertical stroke
 operator.\u2916.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail
 operator.\u2917.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail with vertical stroke
 operator.\u2918.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail with double vertical stroke
 operator.\u2919.infix = lspace:5 rspace:5 accent # leftwards arrow-tail
 operator.\u291A.infix = lspace:5 rspace:5 accent # rightwards arrow-tail
 operator.\u291B.infix = lspace:5 rspace:5 accent # leftwards double arrow-tail
@@ -722,49 +722,49 @@ operator.\u2946.infix = lspace:5 rspace:
 operator.\u2947.infix = lspace:5 rspace:5 accent # rightwards arrow through x
 operator.\u2948.infix = lspace:5 rspace:5 accent # left right arrow through small circle
 operator.\u2949.infix = lspace:5 rspace:5 # upwards two-headed arrow from small circle
 operator.\u294A.infix = lspace:5 rspace:5 accent # left barb up right barb down harpoon
 operator.\u294B.infix = lspace:5 rspace:5 accent # left barb down right barb up harpoon
 operator.\u294C.infix = lspace:5 rspace:5 # up barb right down barb left harpoon
 operator.\u294D.infix = lspace:5 rspace:5 # up barb left down barb right harpoon
 operator.\u294E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightVector;
-operator.\u294F.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpDownVector;
+operator.\u294F.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpDownVector;
 operator.\u2950.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftRightVector;
-operator.\u2951.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpDownVector;
+operator.\u2951.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpDownVector;
 operator.\u2952.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVectorBar;
 operator.\u2953.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVectorBar;
-operator.\u2954.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpVectorBar;
-operator.\u2955.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownVectorBar;
+operator.\u2954.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpVectorBar;
+operator.\u2955.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownVectorBar;
 operator.\u2956.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVectorBar;
 operator.\u2957.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVectorBar;
-operator.\u2958.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpVectorBar;
-operator.\u2959.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownVectorBar;
+operator.\u2958.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpVectorBar;
+operator.\u2959.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownVectorBar;
 operator.\u295A.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeVector;
 operator.\u295B.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeVector;
-operator.\u295C.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpTeeVector;
-operator.\u295D.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownTeeVector;
+operator.\u295C.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpTeeVector;
+operator.\u295D.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownTeeVector;
 operator.\u295E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftTeeVector;
 operator.\u295F.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightTeeVector;
-operator.\u2960.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpTeeVector;
-operator.\u2961.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownTeeVector;
+operator.\u2960.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpTeeVector;
+operator.\u2961.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownTeeVector;
 operator.\u2962.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above leftwards harpoon with barb down
 operator.\u2963.infix = lspace:5 rspace:5 # upwards harpoon with barb left beside upwards harpoon with barb right
 operator.\u2964.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above rightwards harpoon with barb down
 operator.\u2965.infix = lspace:5 rspace:5 # downwards harpoon with barb left beside downwards harpoon with barb right
 operator.\u2966.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above rightwards harpoon with barb up
 operator.\u2967.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb down above rightwards harpoon with barb down
 operator.\u2968.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above leftwards harpoon with barb up
 operator.\u2969.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb down above leftwards harpoon with barb down
 operator.\u296A.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above long dash
 operator.\u296B.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb down below long dash
 operator.\u296C.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above long dash
 operator.\u296D.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb down below long dash
-operator.\u296E.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpEquilibrium;
-operator.\u296F.infix = lspace:2 rspace:2 stretchy direction:vertical # &ReverseUpEquilibrium;
+operator.\u296E.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpEquilibrium;
+operator.\u296F.infix = lspace:5 rspace:5 stretchy direction:vertical # &ReverseUpEquilibrium;
 operator.\u2970.infix = lspace:5 rspace:5 # &RoundImplies;
 operator.\u2971.infix = lspace:5 rspace:5 accent # equals sign above rightwards arrow
 operator.\u2972.infix = lspace:5 rspace:5 accent # tilde operator above rightwards arrow
 operator.\u2973.infix = lspace:5 rspace:5 accent # leftwards arrow above tilde operator
 operator.\u2974.infix = lspace:5 rspace:5 accent # rightwards arrow above tilde operator
 operator.\u2975.infix = lspace:5 rspace:5 accent # rightwards arrow above almost equal to
 operator.\u2976.infix = lspace:5 rspace:5 accent # less-than above leftwards arrow
 operator.\u2977.infix = lspace:5 rspace:5 accent # leftwards arrow through less-than
@@ -899,19 +899,19 @@ operator.\u29F7.infix = lspace:4 rspace:
 operator.\u29F8.infix = lspace:3 rspace:3 # big solidus
 operator.\u29F9.infix = lspace:3 rspace:3 # big reverse solidus
 operator.\u29FA.infix = lspace:3 rspace:3 # double plus
 operator.\u29FB.infix = lspace:3 rspace:3 # triple plus
 operator.\u29FC.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left-pointing curved angle bracket
 operator.\u29FD.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right-pointing curved angle bracket
 operator.\u29FE.infix = lspace:4 rspace:4 # tiny
 operator.\u29FF.infix = lspace:4 rspace:4 # miny
-operator.\u2A00.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigodot;
-operator.\u2A01.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigoplus;
-operator.\u2A02.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigotimes;
+operator.\u2A00.prefix = lspace:1 rspace:2 largeop movablelimits symmetric # &bigodot;
+operator.\u2A01.prefix = lspace:1 rspace:2 largeop movablelimits symmetric # &bigoplus;
+operator.\u2A02.prefix = lspace:1 rspace:2 largeop movablelimits symmetric # &bigotimes;
 operator.\u2A04.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &biguplus;
 operator.\u2A06.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &bigsqcup;
 operator.\u2A0B.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # summation with integral
 operator.\u2A0C.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # quadruple integral operator
 operator.\u2A0D.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # finite part integral
 operator.\u2A0E.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with double stroke
 operator.\u2A0F.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral average with slash
 operator.\u2A10.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # circulation function
@@ -940,17 +940,17 @@ operator.\u2A26.infix = lspace:4 rspace:
 operator.\u2A27.infix = lspace:4 rspace:4 # plus sign with subscript two
 operator.\u2A28.infix = lspace:4 rspace:4 # plus sign with black triangle
 operator.\u2A29.infix = lspace:4 rspace:4 # minus sign with comma above
 operator.\u2A2A.infix = lspace:4 rspace:4 # minus sign with dot below
 operator.\u2A2B.infix = lspace:4 rspace:4 # minus sign with falling dots
 operator.\u2A2C.infix = lspace:4 rspace:4 # minus sign with rising dots
 operator.\u2A2D.infix = lspace:4 rspace:4 # plus sign in left half circle
 operator.\u2A2E.infix = lspace:4 rspace:4 # plus sign in right half circle
-operator.\u2A2F.infix = lspace:2 rspace:2 # &Cross;
+operator.\u2A2F.infix = lspace:4 rspace:4 # &Cross;
 operator.\u2A30.infix = lspace:4 rspace:4 # multiplication sign with dot above
 operator.\u2A31.infix = lspace:4 rspace:4 # multiplication sign with underbar
 operator.\u2A32.infix = lspace:4 rspace:4 # semidirect product with bottom closed
 operator.\u2A33.infix = lspace:4 rspace:4 # smash product
 operator.\u2A34.infix = lspace:4 rspace:4 # multiplication sign in left half circle
 operator.\u2A35.infix = lspace:4 rspace:4 # multiplication sign in right half circle
 operator.\u2A36.infix = lspace:4 rspace:4 # circled multiplication sign with circumflex accent
 operator.\u2A37.infix = lspace:4 rspace:4 # multiplication sign in double circle
